scores-src/src/common/components/Clock.tsx

```diff
@@ -23,8 +24,6 @@ export function RenderClock({
     return () => clearInterval(interval);
   }, [clock.state]);
   return (
-    <span style={{ fontVariantNumeric: "tabular-nums" }}>
-      {formatMMSSMS(time, precisionMs, precisionHigh)}
-    </span>
+    <Title order={3}>{formatMMSSMS(time, precisionMs, precisionHigh)}</Title>
```

Comment:
You'll want to preserve the (JSX equivalent of the) CSS `font-variant-numeric: tabular-nums` - that ensures the numbers keep a consistent letter spacing and avoids it jumping around when it changes.

Reply:
oh, good call - amended and pushed